src/lib/modernExtend.ts

```diff
@@ -151,108 +374,274 @@ export function onOff(args?: OnOffArgs): ModernExtend {
-        toZigbee = [tz.metering_power, tz.currentsummdelivered];
-        delete configureLookup.haElectricalMeasurement;
-    } else if (args.cluster === 'electrical') {
-        exposes = [e.power().withAccess(ea.STATE_GET), e.voltage().withAccess(ea.STATE_GET), e.current().withAccess(ea.STATE_GET)];
-        fromZigbee = [fz.electrical_measurement];
-        toZigbee = [tz.electrical_measurement_power, tz.acvoltage, tz.accurrent];
-        delete configureLookup.seMetering;
-    }
+    const fromZigbee: Fz.Converter[] = [
+        {
+            cluster: 'genOnOff',
+            type: ['commandOn', 'commandOff', 'commandOffWithEffect', 'commandToggle'],
+            convert: (model, msg, publish, options, meta) => {
+                if (hasAlreadyProcessedMessage(msg, model)) return;
+                const payload = {action: postfixWithEndpointName(actionPayloadLookup[msg.type], msg, model, meta)};
+                addActionGroup(payload, msg, model);
+                return payload;
+            },
+        },
+    ];
 
-    const configure: Configure = async (device, coordinatorEndpoint, logger) => {
-        for (const [cluster, properties] of Object.entries(configureLookup)) {
-            for (const endpoint of getEndpointsWithInputCluster(device, cluster)) {
-                const items: ReportingConfig[] = [];
-                for (const property of Object.values(properties)) {
-                    // In case multiplier or divisor was provided, use that instead of reading from device.
-                    if (property.forced) {
-                        endpoint.saveClusterAttributeKeyValue(cluster, {
-                            [property.divisor]: property.forced.divisor ?? 1,
-                            [property.multiplier]: property.forced.multiplier ?? 1,
-                        });
-                        endpoint.save();
-                    } else {
-                        await endpoint.read(cluster, [property.divisor, property.multiplier]);
-                    }
+    const result: ModernExtend = {exposes, fromZigbee, isModernExtend: true};
 
-                    const divisor = endpoint.getClusterAttributeValue(cluster, property.divisor);
-                    assertNumber(divisor, property.divisor);
-                    const multiplier = endpoint.getClusterAttributeValue(cluster, property.multiplier);
-                    assertNumber(multiplier, property.multiplier);
-                    let change: number | [number, number] = property.change * (divisor / multiplier);
-                    // currentSummDelivered data type is uint48, so reportableChange also is uint48
-                    if (property.attribute === 'currentSummDelivered') change = [0, change];
-                    items.push({attribute: property.attribute, min: '10_SECONDS', max: 'MAX', change});
+    if (args.bind) {
+        result.configure = async (device, coordinatorEndpoint, logger) => {
+            if (args.endpointNames) {
```

Comment:
This `if/else` is duplicated quite some times, please move into a function.

Reply:
Added new `setupConfigureForBinding` for this